src/cart/SConscript

```diff
@@ -91,11 +81,8 @@ def scons():
 
     denv = env.Clone(LIBS=[])
 
-    # Generate cart utility shared objects
-    build_utility_shared_obj(denv)
-
     denv.AppendUnique(LIBS=['gurt'])
-    prereqs.require(denv, 'mercury')
+    prereqs.require(denv, 'mercury', 'protobufc')
```

Comment:
is protobuf needed here?

Reply:
`protobufc` isn't needed there.  It's only needed in [src/cart/utils](https://github.com/daos-stack/daos/pull/6219/files/74092f55cc5e7ac5e09bfd83f0d7f80c39926787#diff-29b3cefb1e87c5dce6289b7576dee25213ca370580650564b5a7f70e1ed0e661R32).